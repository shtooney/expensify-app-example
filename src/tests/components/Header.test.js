import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';

test('should render Header correctly', () => {

    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // sees how many H1 tags exist --> only 1 here
    //expect(wrapper.find('h1').length).toBe(1);
    //expect(wrapper.find('h1').text()).toBe('Expensify');
    // etc info
});