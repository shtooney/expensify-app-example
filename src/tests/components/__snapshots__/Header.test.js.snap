// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
  </div>
  <div>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/create"
    >
      Expense
    </NavLink>
  </div>
</header>
`;
